README.md

```diff
@@ -27,6 +27,9 @@ crates.
   handle the next request, `tower-buffer` stores the request in an internal
   queue ([docs][tbuf-docs]).
 
+* [`tower-direct-service`]: Similar to the `Service` trait, but allows for the
```

Comment:
Probably don't want to add this just now since we might not be going through with this idea.